<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyManagerTab</class>
 <widget class="QWidget" name="KeyManagerTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vboxMain">
   <item>
    <widget class="MessageWidgetStack" name="msgWidget" native="true"/>
   </item>
   <item>
    <widget class="QTreeViewOpt" name="treeKeyStore">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hboxButtons">
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>I&amp;mport</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspcButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionImportWiiKeysBin">
   <property name="text">
    <string>Wii keys.bin</string>
   </property>
  </action>
  <action name="actionImportWiiUOtpBin">
   <property name="text">
    <string>Wii U otp.bin</string>
   </property>
  </action>
  <action name="actionImport3DSboot9bin">
   <property name="text">
    <string>3DS boot9.bin</string>
   </property>
  </action>
  <action name="actionImport3DSaeskeydb">
   <property name="text">
    <string>3DS aeskeydb.bin</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTreeViewOpt</class>
   <extends>QTreeView</extends>
   <header>config/QTreeViewOpt.hpp</header>
  </customwidget>
  <customwidget>
   <class>MessageWidgetStack</class>
   <extends>QWidget</extends>
   <header>config/MessageWidgetStack.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
